packages/block-library/src/embed/editor.scss

```diff
@@ -10,7 +10,7 @@
 	// Apply a min-width, or the embed can collapse when floated.
 	// Instagram widgets have a min-width of 326px, so go a bit beyond that.
 	@include break-small() {
-		min-width: 360px;
+		width: 100%;
```

Comment:
Is this width actually necessary? How does it behave without it?

Reply:
You know, not only I removed that and it works just find I also removed an override rule for the placeholder and everything is still fine ;)